### PR TITLE
Avatar1

### PR DESCRIPTION
**Pull request by therajatg**
Please review the avatar component

### CODE REVIEW

File: styles/main-style.css [hunk 1]
```diff
@@ -34,32 +34,43 @@ body{
     font-family: var(--general-font);
     display: flex;
     flex-direction: column;
-    row-gap: 10px;
```
Comment (vaibhav-mahalle) [inline styles/main-style.css]: remove the commented code if not necessary.

File: styles/main-style.css [hunk 2]
```diff
@@ -34,32 +34,43 @@ body{
...
 .content{
     display: flex;
     flex-direction: column;
     padding: 2rem;
+    row-gap: 10px;
     /* justify-content: center; */
 }
 
-.language-html{
+/* .language-html{
```
Comment (vaibhav-mahalle) [inline styles/main-style.css]: same as above

File: docs/index.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+                <li><a href="image.html">Image</a></li>
+                <li><a href="input.html">Input</a></li>
+                <li><a href="text-utilities.html">Text Utilities</a></li>
+                <li><a href="lists.html">Lists</a></li>
+                <li><a href="navigation.html">Navigation</a></li>
+                <li><a href="modal.html">Modal</a></li>
+                <li><a href="rating.html">Rating</a></li>
+                <li><a href="toast.html">Toast/Notify/Snackbar</a></li>
+                <li><a href="simplified-grid.html">Simplified Grid</a></li>
+                <!-- <li><a href="navigation.html">Navigation</a></li>
```
Comment (vaibhav-mahalle) [inline docs/index.html:39]: remove unnecessary code

**Review by vaibhav-mahalle — COMMENTED** [review]
Everything else looks good.

File: docs/avatar.html [hunk 4]
```diff
@@ -5,24 +5,120 @@
...
+            </div>
+    
+            <div>
+            <p>1. Basic avatar icon:</p>
+            <div class="avatar-container basic">
+                <img class="avatar avatar-one" src="/pics/avatar/avatar1.jpg" alt="">
+                <img class="avatar avatar-two" src="/pics/avatar/avatar2.jpg" alt="">
+                <img class="avatar avatar-three" src="/pics/avatar/avatar3.jpg" alt="">
+                <img class="avatar avatar-four" src="/pics/avatar/avatar4.jpg" alt="">
+                <img class="avatar avatar-five" src="/pics/avatar/avatar5.jpg" alt="">
```
Comment (devananth) [inline docs/avatar.html]: Add values to `.alt`

Comment (therajatg) [reply docs/avatar.html]: I am removing the .alt itself. Thanks!

File: styles/main-style.css (hunk 1 above)
Comment (therajatg) [reply styles/main-style.css]: Will remove the commented part.

Comment (therajatg) [reply styles/main-style.css]: Vaibhav, I am not sure but maybe you have to resolve the conservation.

File: docs/avatar.html (hunk 4 above)
Comment (therajatg) [reply docs/avatar.html]: Devananth, I am not sure but maybe you have to resolve the conservation.

Comment (devananth) [reply docs/avatar.html]: I think `.alt` is important , it helps in **accessiblity**. I would recommend you to add it.

Comment (therajatg) [reply docs/avatar.html]: The required alt attribute specifies an alternate text for an image if the image cannot be displayed. The alt attribute provides alternative information for an image if a user for some reason cannot view it. 
As of now I can't think of any alternative to use in place of those images.
I'll keep it in mind if I observe that those images are slow to load. 

Thanks for letting me know about alt (I did not knew its purpose before)

File: docs/index.html (hunk 3 above)
Comment (therajatg) [reply docs/index.html:39]: done

File: docs/avatar.html (hunk 4 above)
Comment (devananth) [reply docs/avatar.html]: Okay , Proceed !

Comment (therajatg) [reply docs/avatar.html]: 👍

**Comment by therajatg** [comment]
Merging

**Pull request merged**